src/XML/src/FMI3/fmi3_xml_model_structure.c

```diff
@@ -253,21 +253,21 @@ static int fmi3_xml_parse_dependencies(fmi3_xml_parser_context_t* context,
     }
     if (listInd) {
         const char* cur = listInd;
-        int ind;
+        long ind;
```

Comment:
While formally an unsigned int, we would no longer be able to catch e.g., negative values here.